replace hpcloud/tail by fork

nxadm opened the pull request:
hpcloud/tail has been unmaintained for 1 year and half. nxadm/tail is a maintained repo where all the valid PRs in the queue were merged. The code where fixed when it was no longer building on Travis and on some UNIX architectures. There are often new releases following the go modules setup.

--

Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo: https://github.com/nxadm/tail
- godoc.org: https://godoc.org/github.com/nxadm/tail
- goreportcard.com:
- coverage service link ([gocover](http://gocover.io/), [coveralls](https://coveralls.io/) etc.)


Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [ x] I have added my package in alphabetical order.
- [ x] I have an appropriate description with correct grammar.
- [x ] I know that this package was not listed before.
- [ x] I have added godoc link to the repo and to my pull request.
- [ ] I have added coverage service link to the repo and to my pull request.
- [ ] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

avelino commented:
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body content check:
- godoc.org: True
- goreportcard.com: False
- coverage: True

panjf2000 commented:
We can't judge a repo unmaintained just based on its recent inactive commits, a repo would be considered unmaintained when it is archived or getting an announcement from the author.

nxadm commented:
@panjf2000, fair enough. However, since Activestate was taken over by HP there is no activity whatsoever: no support for go modules, failing builds, no responses to issues or PRs. Therefore I don't expect the repo to be archived. I will keeping importing the PRs and addressing the issues in the meantime. Thank you for the consideration, we will re-evaluate in the future.
